### PR TITLE
Prebid Core: Add ttl buffer to videoCache.js

### PR DESCRIPTION
**Pull request by patmmccann**
Re-implements #3163 with a buffer

### CODE REVIEW

File: modules/dgkeywordRtdProvider.js [hunk 1]
```diff
@@ -25,6 +25,15 @@ export function getDgKeywordsAndSet(reqBidsConfigObj, callback, moduleConfig, us
...
   const adUnits = reqBidsConfigObj.adUnits || getGlobal().adUnits;
+  callback = (function(cb) {
+    let done = false;
+    return function () {
+      if (!done) {
+        done = true;
+        return cb.apply(this, arguments);
+      }
+    }
+  })(callback);
```
Comment (ncolletti) [inline modules/dgkeywordRtdProvider.js:36]: Should this change be included?

File: modules/sonobiBidAdapter.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ const OUTSTREAM_REDNERER_URL = 'https://mtrx.go.sonobi.com/sbi_outstream_rendere
 
 export const spec = {
   code: BIDDER_CODE,
+  gvlid: 104,
```
Comment (ncolletti) [inline modules/sonobiBidAdapter.js:14]: This change seems unrelated to the PR

File: src/videoCache.js [hunk 3]
```diff
@@ -63,11 +63,11 @@ function wrapURI(uri, impUrl) {
 function toStorageRequest(bid, {index = auctionManager.index} = {}) {
   const vastValue = bid.vastXml ? bid.vastXml : wrapURI(bid.vastUrl, bid.vastImpUrl);
   const auction = index.getAuction(bid);
-
+  const ttlWithBuffer = Number(bid.ttl) + 15;
```
Comment (ncolletti) [inline src/videoCache.js:66]: Could we initialize the buffer amount to a descriptively named const?

File: modules/dgkeywordRtdProvider.js (hunk 1 above)
Comment (patmmccann) [reply modules/dgkeywordRtdProvider.js:36]: When i merged in master in the last commit all the recent changes from master came in, I'll revert

File: src/videoCache.js (hunk 3 above)
Comment (patmmccann) [reply src/videoCache.js:66]: Yeah

**Review by ncolletti — APPROVED** [review]
lgtm

**Comment by ncolletti** [comment]
@ChrisHuie realizing this is a Core change, it's on the smaller end and I'm confident in the change but a second reviewer approval might help before merging.

**Review by ChrisHuie — APPROVED** [review]
